### PR TITLE
docs: add setup instructions for VS Code

### PR DESCRIPTION
**Pull request by towynlin**
As promised here https://bristlemouth.discourse.group/t/setting-up-in-vs-code/261/2?u=zachary

### CODE REVIEW

**Review by russelldeguzman — APPROVED** [review]

File: docs/vscode-setup.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+We're going to create and edit files in there.
+
+
+## 1. CMake and the conda env
+
+- Install the CMake Tools extension from Microsoft.
+- If you don't yet have the `code` command in your terminal PATH to open the VS Code app, then:
+    - In VS Code, hit Cmd-Shift-P and choose "Shell Command: Install 'code' command in PATH"
+    - In order to get VS Code to work in your conda environment, you'll need to **always open it from a terminal where you're already in the conda environment**. For example, open terminal, cd to the `bm_protocol` directory, `conda activate bristlemouth`, then `code .` (with the dot argument) meaning "open the current directory in VS Code". Go ahead and quit VS Code and open it this way now.
+
```
Comment (victorsowa12) [inline docs/vscode-setup.md:24]: have you thought about making a bash script that will do this for you? Then you can make a little icon that lives on the task bar. I've done this for sourcetree(git gui app that didn't work properly until I executed it from terminal).

**Review by victorsowa12 — APPROVED** [review]
Looks good!

File: docs/vscode-setup.md (hunk 1 above)
Comment (towynlin) [reply docs/vscode-setup.md:24]: Ah interesting! For me personally, typing `code .` in the terminal works really well, but that could be a super  helpful tip for many people Want to add something to this PR @victorsowa12?

Comment (victorsowa12) [reply docs/vscode-setup.md:24]: I would need to do a little test to make sure it works first. I will hold off on adding anything so this PR can go out!

**Pull request merged**